documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc

```diff
@@ -6,53 +6,159 @@
 
-NOTE: No alert notification rules are defined.
+* `strimzi-kafka.json`
+* `strimzi-zookeeper.json`
+* `strimzi-kafka-connect.json`
+* `strimzi-kafka-mirror-maker-2.json`
+* `strimzi-operators.json`
 
-When accessing a dashboard, you can use the `port-forward` command to forward traffic from the Grafana pod to the host.
+The example dashboards are a good starting point for monitoring key metrics, but they do not represent all available metrics.
+You may need to modify the example dashboards or add other metrics, depending on your infrastructure.
 
-For example, you can access the Grafana user interface by:
+After setting up Prometheus and Grafana, you can visualize the Strimzi data on the Grafana dashboards.
 
-. Running `kubectl port-forward svc/grafana 3000:3000`
-. Pointing a browser to `http://localhost:3000`
+NOTE: No alert notification rules are defined.
+
+When accessing a dashboard, you can use the `port-forward` command to forward traffic from the Grafana pod to the host.
 
 NOTE: The name of the Grafana pod is different for each user.
 
 .Procedure
 
-. Access the Grafana user interface using `admin/admin` credentials.
+. Get the details of the Grafana service:
++
+[source,shell]
+----
+kubectl get service grafana
+----
+
++
+For example:
++
+[table,stripes=none]
+|===
+|NAME     |TYPE      |CLUSTER-IP    |PORT(S)
+
+|grafana  |ClusterIP |172.30.123.40 |80/TCP
+|===
++
+Note the port number for port forwarding.
+
+. Use `port-forward` to redirect the Grafana user interface to `localhost:3000`:
++
+[source,shell]
+----
+kubectl port-forward svc/grafana 3000:80
+----
 +
-On the initial view choose to reset the password.
+NOTE: If using the OpenShift web console, you can switch to menu:Networking[Routes], and click the URL for the `grafana` Route.
+
+. Point a web browser to `http://localhost:3000`.
 +
-image::grafana_login.png[Grafana login]
+The Grafana Log In page appears.
 
-. Click the *Add data source* button.
+. Enter your user name and password, and then click btn:[Log In].
 +
-image::grafana_home.png[Grafana home]
+The default Grafana user name and password are both `admin`. After logging in for the first time, you can change the password.
 
-. Add Prometheus as a data source.
+. Add Prometheus as a _data source_.
 +
 * Specify a name
 * Add _Prometheus_ as the type
-* Specify the connection string to the Prometheus server (http://prometheus-operated:9090) in the URL field
-
-. Click *Add* to test the connection to the data source.
+* Specify a Prometheus server URL (http://prometheus-operated:9090)
++
+Save and test the connection when you have added the details.
 +
 image::grafana_prometheus_data_source.png[Add Prometheus data source]
 
-. Click *Dashboards*, then *Import* to open the _Import Dashboard_ window and import the example dashboards (or paste the JSON).
+. From menu:Dashboards[Import], upload the example dashboards or paste the JSON directly.
```

Comment:
Are the `menu` and `btn` tags definitely rendered in the Strimzi docs?

Reply:
Good point. I built with asciidoctor and ccutil and seems okay. If they don't come out okay, I'll adjust